src/components/calculate-watts.js

```diff
@@ -1,56 +1,63 @@
 import React from "react"
+import Slider from './slider';
 
 class CalculateWatts extends React.Component {
-    constructor(props) {
-      super(props);
-      this.state = { watt: '' };
-  
-      this.handleSetLength = this.handleSetLength.bind(this);
-      this.handleSetNumberOfLedsPerMeter = this.handleSetNumberOfLedsPerMeter.bind(this);
-      this.handleSetMaximumPowerDrawPerLed = this.handleSetMaximumPowerDrawPerLed.bind(this);
+    state = {
+      watt: ''
     }
-  
-    handleSetLength(event) {
-      this.setState({ length: event.target.value })
-    }
-  
-    handleSetNumberOfLedsPerMeter(event) {
-      this.setState({ numberOfLedsPerMeter: event.target.value });
-    }
-  
-    handleSetMaximumPowerDrawPerLed(event) {
-      this.setState({ maximumPowerDrawPerLed: event.target.value });
-    }
-  
+
     get wattRequired() {
       return this.state.length * this.state.numberOfLedsPerMeter * this.state.maximumPowerDrawPerLed;
     }
-  
+
+    get canCalculate() {
+      return (
+        this.state.length && this.state.numberOfLedsPerMeter && this.state.maximumPowerDrawPerLed
+      )
+    }
+
     render() {
       return (
         <div>
           <form onSubmit={this.handleSubmit}>
-            <p>
-              <label>
-                How long is your ledstrip (in meter): 
-                <input type="numeric" name="length" onChange={this.handleSetLength} />
-              </label>
-            </p>
-            <p>
-              <label>
-                How many leds per meter:
-                <input type="numeric" name="numberOfLedsPerMeter" onChange={this.handleSetNumberOfLedsPerMeter} />
-              </label>
-            </p>
-            <p>
-              <label>
-                Maximum power draw per led:
-                <input type="numeric" name="maximumPowerDrawPerLed" onChange={this.handleSetMaximumPowerDrawPerLed} />
-              </label>
-            </p>
+            <div>
+              <h3>
+                <span role="img" aria-label="ruler emoji">📏 </span>
+                How long is your LED strip? (in meter)
+              </h3>
+              <Slider
+                max={10}
```

Comment:
This will limit to 10M ledstrips, not sure if this is wanted, otherwise feel free to edit.

Reply:
I'll change it to even more as you can chain 5m led strips. But maybe i'll add a warning once you go above 5m that you will need to connect the power source every 5m or so.